src/Compilers/CSharp/Test/Symbol/Compilation/SemanticModelGetDeclaredSymbolAPITests.cs

```diff
@@ -5194,5 +5194,76 @@ public override void VisitIncompleteMember(IncompleteMemberSyntax node)
                 base.VisitIncompleteMember(node);
             }
         }
+
+        [Fact]
+        public void InvalidParameterWithDefaultValue_Method()
+        {
+            var source =
+@"class Program
+{
+    static void F(int x = 2, = 3) { }
+}";
+            var comp = CreateCompilation(source);
+            var tree = comp.SyntaxTrees[0];
+            var model = comp.GetSemanticModel(tree);
+            var decls = tree.GetCompilationUnitRoot().DescendantNodes().OfType<ParameterSyntax>().ToArray();
+            var symbol1 = model.GetDeclaredSymbol(decls[0]);
+            Assert.Equal("[System.Int32 x = 2]", symbol1.ToTestDisplayString());
+            Assert.Equal(0, ((ParameterSymbol)symbol1).Ordinal);
+            var symbol2 = model.GetDeclaredSymbol(decls[1]);
+            Assert.Equal("[?  = null]", symbol2.ToTestDisplayString());
+            Assert.Equal(1, ((ParameterSymbol)symbol2).Ordinal);
+            Assert.Same(symbol1.ContainingSymbol, symbol2.ContainingSymbol);
+        }
+
+        [Fact]
+        [WorkItem(784401, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/784401")]
+        public void InvalidParameterWithDefaultValue_LocalFunction_01()
+        {
+            var source =
+@"class Program
+{
+    static void Main()
+    {
+        void F(int x, = 3) { }
+    }
+}";
+            var comp = CreateCompilation(source);
+            var tree = comp.SyntaxTrees[0];
+            var model = comp.GetSemanticModel(tree);
+            var decls = tree.GetCompilationUnitRoot().DescendantNodes().OfType<ParameterSyntax>().ToArray();
+            var symbol1 = model.GetDeclaredSymbol(decls[0]);
+            Assert.Equal("System.Int32 x", symbol1.ToTestDisplayString());
+            Assert.Equal(0, ((ParameterSymbol)symbol1).Ordinal);
+            var symbol2 = model.GetDeclaredSymbol(decls[1]);
+            Assert.Equal("[?  = null]", symbol2.ToTestDisplayString());
+            Assert.Equal(1, ((ParameterSymbol)symbol2).Ordinal);
+            Assert.Same(symbol1.ContainingSymbol, symbol2.ContainingSymbol);
+        }
+
+        [Fact]
+        [WorkItem(784401, "https://devdiv.visualstudio.com/DevDiv/_workitems/edit/784401")]
+        public void InvalidParameterWithDefaultValue_LocalFunction_02()
+        {
+            var source =
+@"class Program
+{
+    static void Main()
+    {
+        void F(int x = 2, = 3) { }
```

Comment:
>3 [](start = 28, length = 1)

I think I'm satisfied with this change, but can you please add some verification of other semantic model apis on the initializer here? Like, GetConstantValue or GetTypeInfo?